vpr/src/place/move_utils.h

```diff
@@ -4,6 +4,13 @@
 #include "move_transactions.h"
 #include "compressed_grid.h"
 
+//Run-time flag to control when placer debug information is printed
+//Note : only enables debug output if compiled with VTR_ENABLE_DEBUG_LOGGING is defined
+#ifdef VTR_ENABLE_DEBUG_LOGGING
+extern bool f_placer_debug;
+#endif
```

Comment:
Rather than exposing this as a global variable in the header, I'd suggest wrapping it in an accessor function. something like: 
```cpp
bool placer_debug_enabled(); //In .h
bool placer_debug_enabled() { return f_placer_debug; } //In .cpp
```

Reply:
Agreed.